dist/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
If `dist/` is being gitignored, should this still be showing up in the diff?

Reply:
Yep, they were still in the git cache, but I think my latest commit (`df726bd`) has potentially solved it! We'll get there in the end 😬